src/components/overlay/overlay.js

```diff
@@ -42,6 +42,24 @@ class Overlay extends Component {
     this.props.removePlayerClass(style.overlayActive);
   }
 
+  /**
+   * closeButton
+   * @param {any} props - props
+   * @returns {React$Element | void} close button element
+   * @memberof Overlay
+   */
+  renderCloseButton(props: any): React$Element<any> | void {
+    if (!props.permanent) {
```

Comment:
What is `props.permanent`?

Reply:
it is a new prop i am passing, to determine if the window is permanent (meaning it can't be closed) - so no close button is needed